src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -261,6 +263,14 @@ def buildAndPushContainerImage(credentialsId, registry, imageName, tag) {
   }
 }
 
+def mapArrayToArgs(args){
```

Comment:
should this function replace the logic in the `triggerDeploy` logic to fix the issue with the version not being set, or is it it intended for another purpose?

If it's for the trigger deploy it's missing the `&` when concatenating the key value pair, and would return
`key1=val1 key2=val2` rather than `&key1=val1&key2=val2` from the function

Reply:
It was intended for another purpose.  It was used when I was passing in build arguments to Docker compose.  I removed reference to it from the functions and the readme but forgot to remove the actual function!